modules/osdk-init-resource.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-generating-csvs.adoc
+
+[id="osdk-init-resource_{context}"]
+= Providing templates for required custom resources
```

Comment:
Is it possible to slot in "initialization" or "init" somewhere in the title to connect the title with the actual annotation naming? thanks!

Reply:
Working title was "Initializing required custom resources"; I'll switch back to that. :)